content/account_management/rbac/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Role Based Access Control
+kind: documentation
+aliases:
+  - /guides/rbac
+further_reading:
+- link: "account_management/rbac/log_management/"
+  tag: "Documentation"
+  text: "RBAC for Log Management"
+- link: "account_management/rbac/role_api"
+  tag: "Documentation"
+  text: "Manage Roles and Permissions with the Role API"
+---
+
+<div class="alert alert-warning">
+RBAC is only available for Log Management. This feature is in private beta. Ask your sales representative or customer success manager to have it enabled.
```

Comment:
not support?

Reply:
@estib will need to respond to this one

Reply:
We typically route the request through sales/csms for private beta enable requests, no?

In any case, it would be advantageous for us to have the request go through sales/csms in this case, since that way we're more likely to get the sales/deal-related context when the request comes in.